docs/openapi/components/schemas/common/LEIaddress.yml

```diff
@@ -75,20 +82,19 @@ properties:
     type: string
     $linkedData:
       term: postalCode
-      '@id': https://schema.org/postalCode
+      '@id': https://www.gleif.org/ontology/Base/hasPostalCode
 additionalProperties: false
 required:
   - type
-  - addressLines
+  - firstAddressLine
   - city
-  - region
   - country
-  - postalCode
 example: |-
   {
     "type": ["LEIaddress"],
     "language": "ru",
-    "addressLines": [
+    "firstAddressLine": "29996 Hahn Extension",
+    "additionalAddressLines": [
```

Comment:
@OR13 The whole concept of address **lines** is obsolete when you can use newline in a string.
But Traceability used it (`addressLines`), I just changed it to whatever is in the latest LEI schema.

The best would be for Traceability to use one definition of `Address`, rather than conflicting definitions from several ontologies. This of course pertains to the class as well (`s:address` vs `LEIaddress` ; the latter is wrong because I don't believe it is mapped to https://www.gleif.org/ontology/Base/PhysicalAddress)